cmd/client-s3.go

```diff
@@ -1716,7 +1716,7 @@ func (c *S3Client) Stat(ctx context.Context, opts StatOptions) (*ClientContent,
 
 	nonRecursive := false
 	maxKeys := 1
-	for objectStat := range c.listObjectWrapper(ctx, bucket, path, nonRecursive, opts.timeRef, false, false, false, maxKeys, opts.isZip) {
+	for objectStat := range c.listObjectWrapper(ctx, bucket, path, nonRecursive, opts.timeRef, true, true, false, maxKeys, opts.isZip) {
```

Comment:
Shouldn't the `--versions` parameter be forwarded to here? It seems like this would cause calls without `--versions` to return duplicates?

Reply:
I will check.